xmppserver/src/main/webapp/setup/index.jsp

```diff
@@ -43,12 +43,9 @@
     boolean openfireHomeExists = false;
     Path openfireHome = null;
 
-    // Check for JRE 1.8
-    try {
-        String version = System.getProperty("java.version");
-        jreVersionCompatible = Integer.parseInt(version.split("\\.")[0]) >= 11;
-    }
-    catch (Throwable t) {}
+    // Check for min JRE requirement
+    Runtime.Version MIN_JAVA_VERSION = Runtime.Version.parse("11");
+    jreVersionCompatible = MIN_JAVA_VERSION.compareTo(Runtime.version()) <= 0;
```

Comment:
When a user runs Java 8, this will fail, since the Runtime API was introduced in Java 10.

I suspect other stuff will also fail in Java 8 too though... @guusdk ?